src/vehicle.h

```diff
@@ -1834,6 +1868,8 @@ class vehicle
         point pos;
         // vehicle current velocity, mph * 100
         int velocity = 0;
+        // vehicle exponential average velocity (av = (av + velocity) / 2), mph * 100
```

Comment:
I don't think that it's wrong, but it may be confusing nevertheless:
https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average

![image](https://user-images.githubusercontent.com/2865203/86419884-89672080-bc89-11ea-9b01-a3be7071b53a.png)

I used a simplified version where α = 0.5:

`v_new =  v_prev + 0.5(v_cur - v_prev) =  0.5 v_cur + 0.5 v_prev`

What can I do to make it less confusing? Add a wikipedia link to the formula?

Reply:
just call it a continuous moving average, people will get what you mean.